modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -21,21 +21,35 @@ The following table lists the temporal value types and their supported component
 [options="header", cols="^,^,^,^"]
 |===
 | Type            | Date support | Time support | Time zone support
-| `Date`          | {check-mark} |              |
-| `Time`          |              | {check-mark} | {check-mark}
-| `LocalTime`     |              | {check-mark} |
-| `DateTime`      | {check-mark} | {check-mark} | {check-mark}
-| `LocalDateTime` | {check-mark} | {check-mark} |
-| `Duration`      | `-`          | `-`          | `-`
+| `DATE`          | {check-mark} |              |
+| `LOCAL TIME`     |              | {check-mark} |
+| `ZONED TIME`          |              | {check-mark} | {check-mark}
+| `LOCAL DATETIME` | {check-mark} | {check-mark} |
+| `ZONED DATETIME`      | {check-mark} | {check-mark} | {check-mark}
+| `DURATION`      | `-`          | `-`          | `-`
 |===
 
-`Date`, `Time`, `LocalTime`, `DateTime`, and `LocalDateTime` are _temporal instant_ types.
+`DATE`, `LOCAL TIME`, `ZONED TIME`, `LOCAL DATETIME`, and `ZONED DATETIME` are _temporal instant_ types.
```

Comment:
Does all this renaming affect users in any way? 

1. Do users ever directly interact with type names in Cypher?
2. I suppose drivers will need to be updated to work with the new types. Have they already been updated? What happens if a user is running, say, driver version 5.3 with the latest neo4j with renamed types?

Reply:
+1 on Louise answer

We haven't previously had named types as such, which is why I too believe drivers will be unaffected by this